src/transformers/deepspeed.py

```diff
@@ -33,6 +33,51 @@
 logger = logging.get_logger(__name__)
 
 
+# XXX: Reza - need the rest of the map
+inference_custom_map = dict(
+    electra=dict(ElectraLayer=("output.dense")),
+    roberta=dict(RobertaLayer=("output.dense")),
+    t5=dict(T5Block=("SelfAttention.o", "EncDecAttention.o", "DenseReluDense.wo")),
+)
+
+# XXX: Reza - need the rest of models that are automated
+inference_auto = [
+    "gpt_neo",
```

Comment:
@RezaYazdaniAminabadi, so gpt-j would go here I think

Reply:
Yes, that's true